Makefile

```diff
@@ -162,6 +164,7 @@ install.cni.sudo: gopath
 install:
 	install -d -m 755 $(DESTDIR)/$(BINDIR)
 	install -m 755 bin/buildah $(DESTDIR)/$(BINDIR)/buildah
+	test -z "${SELINUXOPT}" || chcon --verbose -t $(SELINUXTYPE) $(DESTDIR)/$(BINDIR)/buildah
```

Comment:
you are only labelling the /usr/bin/buildah binary but the test run ./bin/buildah
The chcon should also be made on bin/buildah